src/CLU/Microsoft.Azure.Commands.Compute/Extension/CustomScript/GetAzureVMCustomScriptExtensionCommand.cs

```diff
@@ -27,7 +27,7 @@ namespace Microsoft.Azure.Commands.Compute
         DefaultParameterSetName = GetCustomScriptExtensionParamSetName),
     OutputType(
         typeof(VirtualMachineCustomScriptExtensionContext))]
-    [CliCommandAlias("vm customscriptextension ls")]
+    [CliCommandAlias("vm extension script ls")]
```

Comment:
Yes, there is no 'non custom' script, but it's just about whether the original extension name shall be used.

@singhkay Thoughts?

Is it feasible to include dashes in any alias, i.e. ```vm x-y-z-extension ls```?